Client/Frontend/Browser/Playlist/Managers & Cache/PlaylistCacheLoader.swift

```diff
@@ -700,20 +701,40 @@ extension PlaylistWebLoader: WKNavigationDelegate {
     return .allow
   }
 
-  func webView(_ webView: WKWebView, didReceive challenge: URLAuthenticationChallenge, completionHandler: @escaping (URLSession.AuthChallengeDisposition, URLCredential?) -> Void) {
+  nonisolated public func webView(_ webView: WKWebView, respondTo challenge: URLAuthenticationChallenge) async -> (URLSession.AuthChallengeDisposition, URLCredential?) {
```

Comment:
This `nonisolated` is redundant as the parent is not isolated to any specific actor